### PR TITLE
Updated to use new Minikube-maintained KVM driver

### PR DESCRIPTION
**Pull request by braderhart**

### CODE REVIEW

File: installers/linux/archlinux/PKGBUILD [hunk 1]
```diff
@@ -14,7 +15,7 @@ depends=(
 optdepends=(
   'kubectl-bin: to manage the cluster'
   'virtualbox'
-  'docker-machine-kvm'
+  'docker-machine-driver-kvm2'
```
Comment (r2d4) [inline installers/linux/archlinux/PKGBUILD]: Awesome, I see you've added an AUR package for this.  Thanks.

We update this installer automatically during our releases.  Do you mind adding a similar folder for the kvm driver, possibly at `installers/linux/archlinux-drivers` (maybe not the most correct, but so we don't have to move much else around)

And then add a similar hack entry for updating the driver?  If you add minikube-bot as a additional maintainer on AUR, it should be able to update it automatically.

https://github.com/kubernetes/minikube/blob/master/hack/jenkins/release_update_installers.sh#L32-L45

**Comment by r2d4** [comment]
Fixes https://github.com/kubernetes/minikube/issues/2336

File: installers/linux/archlinux/PKGBUILD (hunk 1 above)
Comment (braderhart) [reply installers/linux/archlinux/PKGBUILD]: Working on this now.

File: hack/jenkins/release_update_installers.sh [hunk 2]
```diff
@@ -19,6 +19,7 @@ git config --global user.email "minikube-bot@google.com"
 
 REPLACE_PKG_VERSION=${VERSION_MAJOR}.${VERSION_MINOR}.${VERSION_BUILD}
 REPLACE_MINIKUBE_LINUX_SHA256=$(awk '{ print $1 }' out/minikube-linux-amd64.sha256)
+REPLACE_MINIKUBE_DRIVER_KVM_SHA256=$(awk '{ print $1 }' out/docker-machine-driver-kvm2.sha256)
```
Comment (braderhart) [inline hack/jenkins/release_update_installers.sh]: Can you please confirm that this is the correct approach?

**Review by see5voph — COMMENTED** [review]
Promise her anything, but give her Exxon unleaded.

File: hack/jenkins/release_update_installers.sh (hunk 2 above)
Comment (r2d4) [reply hack/jenkins/release_update_installers.sh]: Yup this should work

**Comment by braderhart** [comment]
Thanks for your help along the way and introducing me to your automated build pipeline. I appreciate the experience.

**Pull request merged**